specification/servicenetworking/resource-manager/Microsoft.ServiceNetworking/preview/2022-10-01-preview/TrafficController.json

```diff
@@ -1102,7 +1102,7 @@
     "FrontendMode": {
       "type": "string",
       "enum": [
-        "public"
+        "Public"
```

Comment:
Is this change going into a new API version? Right now NFVRP serializes this enum with "camelCase" [for all versions](https://msazure.visualstudio.com/One/_git/Networking-nfv?path=/src/NfvRp/Core/Frontend/Models/TrafficController/Enums/TrafficControllerStringEnumConverter.cs&version=GBdevelop2&line=22&lineEnd=25&lineStartColumn=1&lineEndColumn=27&lineStyle=plain&_a=contents). The code is setup so that for future API versions we could serialize with "PascalCase". But if we change the casing in-place, NFVRP will be broken again.